.github/workflows/release.yml

```diff
@@ -0,0 +1,69 @@
+# From: https://github.com/googleapis/google-cloud-go/blob/master/.github/workflows/release.yaml
+# This workflow creates release PRs and tags the top level release for
+# the top level repository:
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - id: release-pr # Open the release PR from a fork.
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          # token: ${{ secrets.FORKING_TOKEN }}
+          token: ${{ secrets.FORKING_TOKEN }}
+          release-type: go
```

Comment:
For a simple `go` repository, I think you might want to use the `simple` release-type, we don't currently have a generic go releaser.

`simple` will use a version.txt, maintain a changelog, and create dist-tags for you.

Reply:
I think we now have a `go` releaser with:

https://github.com/googleapis/release-please/pull/890

It's undocumented, but I think that will work?

Reply:
I think we have the generic `go` release type now.